src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx

```diff
@@ -42,23 +51,20 @@ const FairHeader: React.FC<FairHeaderProps> = ({ fair, ...rest }) => {
     <Box {...rest}>
       <FairHeaderImage fair={fair} />
 
-      <CSSGrid
-        mt={[2, 4]}
-        gridAutoFlow="row"
-        gridColumnGap={3}
-        gridRowGap={2}
-        gridTemplateColumns={["repeat(1, 1fr)", `repeat(${columnCount}, 1fr)`]}
-        textAlign={columnCount === 1 ? "center" : undefined}
-      >
-        <Box>
-          <Text as="h1" variant="largeTitle">
-            {fair.name}
-          </Text>
+      <GridColumns mt={[2, 4]}>
```

Comment:
Should we deprecate `CSSGrid` over in Palette?

Reply:
It's still useful as a generic kind of CSSGrid when u need to step outside of our 12 column layout. What we should deprecate is `Grid` though, which uses bootstrap grid under the hood.